docs/source/how_to_use.rst

```diff
@@ -7,9 +7,13 @@ Getting Started
   :ref:`repo-structure` )
 - Execute ``runway init`` to create a ``runway.yml`` file at the root of the
   environment (e.g. at the repo root)
+    - If runway was installed from npm, you many need to add
+      ``$projectdir/node_modules/bin`` to your ``PATH`` or supply the
+      relative path to use the runway CLI.
```

Comment:
Let's drop this note -- I like the intent of it, but we'll have a few methods for install and it'll be more straightforward to just assume that the command is available on the path

Reply:
removed note